app/views/helpers/PointerPatterns.scala

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.helpers
+
+import models.{ExportsDeclaration, Pointer, PointerSection}
+import models.PointerSectionType.{FIELD, SEQUENCE}
+import models.declaration.ExportItem
+
+import scala.util.Try
+
+object PointerPatterns {
+  val pointerToDucr = "declaration.consignmentReferences.ucr"
+
+  val expandAdditionalActors = (p: Pointer, orig: ExportsDeclaration, amend: ExportsDeclaration) =>
+    Seq(Pointer(p.sections :+ PointerSection("eori", FIELD)), Pointer(p.sections :+ PointerSection("type", FIELD)))
+
+  val expandCusCode = (p: Pointer, orig: ExportsDeclaration, amend: ExportsDeclaration) =>
+    Seq(Pointer(p.sections :+ PointerSection("cusCode", FIELD)))
+
+  val expandConsignorDetails = (p: Pointer, orig: ExportsDeclaration, amend: ExportsDeclaration) =>
+    Seq(Pointer(p.sections :+ PointerSection("eori", FIELD)))
+
+  val expandDetails = (p: Pointer, orig: ExportsDeclaration, amend: ExportsDeclaration) => Seq(Pointer(p.sections :+ PointerSection("eori", FIELD)))
```

Comment:
What's this for? I do not see differences with `expandConsignorDetails`

Reply:
Yes good spot, looks like we can just reuse the generic `expandDetails` function.